Utils/Dataflow/pyDKB/common/misc.py

```diff
@@ -0,0 +1,49 @@
+"""
+pyDKB.common.misc
+
+Miscellanious utility functions.
+"""
+
+import sys
+import inspect
+from datetime import datetime
+
+from types import logLevel
+
+# Datetime format for log messages
+DTFORMAT = '%Y-%m-%d %H:%M:%S'
+
+
+def log(message, level=logLevel.INFO, *args):
+    """ Output log message with given log level.
+
+    :param message: message to output
+    :type message: str
+    :param level: log level of the message
+    :type level: ``pyDKB.common.types.logLevel`` member
+    :param *args: additional prefixes (will be output between log
+                  level prefix and message body)
+    :type *args: str
+    """
+    if not logLevel.hasMember(level):
+        log("Unknown log level: %s" % level, logLevel.WARN)
+        level = logLevel.INFO
+    if type(message) == list:
+        lines = message
+    else:
+        lines = message.splitlines()
```

Comment:
Maybe `str(message).splitlines()`? This is already done for prefixes `*args` and each member of `lines` if `message` is a `list`:

```
>>> log('abc')
2019-08-26 14:55:21 (INFO) (main) abc
>>> log('abc', logLevel.INFO, str)
2019-08-26 14:55:40 (INFO) (<type 'str'>) abc
>>> log(['abc', str])
2019-08-26 14:55:55 (INFO) (main) abc
(==) <type 'str'>
>>> log(str)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "pyDKB/common/misc.py", line 34, in log
    lines = message.splitlines()
TypeError: descriptor 'splitlines' of 'str' object needs an argument
>>> log(int)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "pyDKB/common/misc.py", line 34, in log
    lines = message.splitlines()
AttributeError: type object 'int' has no attribute 'splitlines'

```

This will also simplify some cases of usage because one wouldn't need to convert to `str` each time. Here:
https://github.com/PanDAWMS/dkb/pull/274/files#diff-9be9001acb47b76c6c21745268c1c088R37
https://github.com/PanDAWMS/dkb/pull/274/files#diff-50722d7919f4fc2cf73626855f67350aR109

Reply:
Done: 5ada0e6f623391d551790ef59a94f89a62d9eb7c